src/Core/src/Handlers/Layout/LayoutExtensions.cs

```diff
@@ -1,59 +1,39 @@
 ﻿using System;
-using System.Collections.Generic;
+using System.Linq;
 
 namespace Microsoft.Maui.Handlers
 {
 	internal static class LayoutExtensions
 	{
-		record ViewAndIndex(IView View, int Index);
-
-		class ZIndexComparer : IComparer<ViewAndIndex>
-		{
-			public int Compare(ViewAndIndex? x, ViewAndIndex? y)
```

Comment:
It appears that a regular `OrderBy()` is the same as this logic. Let me know if I'm missing something, thanks!

Reply:
I found other places where performance matters, that used `OrderBy()`:

https://github.com/dotnet/msbuild/blob/c2ec5c98a76a5496a36025b3f5e790cca6ea3b2f/src/Build/Evaluation/LazyItemEvaluator.cs#L511-L515

This is used when MSBuild evaluates a project, for every type of item group in a build. It would be called when VS loads a project.